packages/slicemachine-core/src/filesystem/authConfig.ts

```diff
@@ -23,75 +18,50 @@ export function createDefaultAuthConfig(directory?: string): AuthConfig {
   return DEFAULT_CONFIG;
 }
 
-/**
- * A function that retrieve the configuration file or create it if it doesn't exist.
- * @param {string} [configPath] - the path to the configuration file
- * @returns {Config} - the config object.
- */
 export function getOrCreateAuthConfig(directory?: string): AuthConfig {
   const configPath = PrismicConfigPath(directory);
   if (!Files.exists(configPath)) return createDefaultAuthConfig(directory);
 
   const conf = Files.readJson(configPath);
-  const completeConf: AuthConfig = { ...DEFAULT_CONFIG, ...conf };
-  return completeConf;
+  return { ...DEFAULT_CONFIG, ...conf } as AuthConfig;
 }
 
-/**
- * A function that update the configuration file
- * @param {string} [configPath] - the path to the configuration file
- * @param {Partial<Config>} [data] - configuration attributes to change
- * @returns {void} nothing
- */
-export function updateAuthConfig(
-  data: Partial<AuthConfig>,
+export function removeAuthConfig(directory?: string): void {
+  const configPath = PrismicConfigPath(directory);
+  return Files.remove(configPath);
+}
+
+export function setAuthConfig(
+  cookies: ReadonlyArray<string> = [],
+  base?: string,
   directory?: string
 ): void {
+  const currentConfig = getOrCreateAuthConfig(directory);
+  const formattedCookies = formatCookies(cookies);
   const configPath = PrismicConfigPath(directory);
-  const oldConfig = getOrCreateAuthConfig(directory);
+
+  const newConfig = {
+    cookies: formattedCookies,
+    base: !!base ? base : currentConfig.base,
```

Comment:
Base is a `string | undefined`, you can just have
```suggestion
    base: base ? base : currentConfig.base,
```

or 

```suggestion
    base: base || currentConfig.base,
```